README.md

```diff
@@ -1 +1,51 @@
 README for Dog-Parks
+
+Title: Dog Parks
+
+Description: Welcome to Dog Parks! This app helps users find dog parks in their area and gives an overview of the features of the park and reviews from other users.
+
+Authors: Alex Nunan, Hilary Gould, Kriss Strikis, Scrith Prosper
+
+Site: https://localhost:300/dog-parks
+
+Technologies:
+- Javascript
+- Express
+- React
+- Foundations CSS
+
+User Stories:
```

Comment:
a readme is providing details to others about what to know about this app - does not need your user stories, as that goes in your planning. there is a README resource in the BT weeks curriculum to provide some insight of what to put in here
https://learn.launchacademy.com/teams/admins/curricula/on-campus-boston-40/lesson_groups/weeks_9_&_10:_final_project/lessons/writing-a-readme-file